<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="1" failures="0" skipped="0" tests="1" time="0.286" timestamp="2024-07-04T09:45:43.901306" hostname="OLI-OMEN"><testcase classname="" name="tests.test_grobid" time="0.000"><error message="collection failure">ImportError while importing test module '/media/olivier/ssd1T/dev/kairntech/pyconverters_grobid/tests/test_grobid.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../../.pyenv/versions/3.8.2/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_grobid.py:5: in &lt;module&gt;
    import pandas as pd
E   ModuleNotFoundError: No module named 'pandas'</error></testcase></testsuite></testsuites>